.travis.yml

```diff
@@ -0,0 +1,50 @@
+# .travis.yml
+
+language: python
+sudo: true
+
+env:
+  - LUA="lua=5.1"
+  - LUA="lua=5.2"
+  - LUA="lua=5.3"
+  - LUA="luajit=2.0"
+  - LUA="luajit=2.1"
```

Comment:
OpenWRT only ships lua 5.1.5 so we can remove all the other lua versions because they just slow down build time, add more overhead for travis-ci and don't add any value, this is the output of `lua -v` on OpenWRT 18.0.6:

```
Lua 5.1.5  Copyright (C) 1994-2012 Lua.org, PUC-Rio (double int32)
```

Reply:
ok i added lua 5.1.5